packages/api/src/external/carequality/__tests__/register-facility.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import { faker } from "@faker-js/faker";
+import { Coordinates } from "@metriport/core/domain/address";
+import { USState } from "@metriport/api-sdk";
+import { AddressWithCoordinates } from "@metriport/core/domain/location-address";
+
+const coordinates: Coordinates = {
+  lat: faker.location.latitude(),
+  lon: faker.location.longitude(),
+};
+
+export const addressWithCoordinates: AddressWithCoordinates = {
+  addressLine1: faker.location.streetAddress(),
+  addressLine2: faker.location.secondaryAddress(),
+  city: faker.location.city(),
+  state: USState.CA,
+  zip: faker.location.zipCode(),
+  country: "USA",
+  coordinates,
+};
```

Comment:
It'd be good to have these as functions, so
1. one could build `addressWithCoordinates` w/ a specific state w/o impacting other test runs
2. we add some controlled randomness to the tests

Can refer to `makePatient` and similar.

Reply:
Removed this file (was copied from the previous implementation of these tests) -- we already had functions defined in domain/medical/__tests__.